flows/index.md

```diff
@@ -0,0 +1,128 @@
+
+***Warning:** If too many tasks are scheduled in a single phase, you may exceed the available Storage job slots, causing delays in your flow’s execution. Limiting the number of concurrent component jobs 
+to 10 is recommended to avoid reaching Storage capacity limits. You can, of course, configure your flows to execute more jobs in parallel. Keboola will then concurrently execute the jobs to the maximum 
+extent possible based on available resources.*
+
+You can also set up parallelization **among components**, directly in the component's UI for [row-based components](/components/#configuration-rows) like database extractors using the same credentials to run multiple tables concurrently.
+
+{% include tip.html title="Execute Individual Configurations" content="
+Run configurations individually in Keboola Flows for more efficient workflow management. You can fine-tune which rows to run in advanced settings for greater 
+control.
+
+Use the menu next to each configuration row in the UI to execute specific configurations as needed, optimizing time and resources.
+" %}
+
+### Parallel limits
+Component jobs don't have a strict parallel limit, but [Storage jobs](/storage/jobs/) do. They are typically capped at 10 parallel jobs but the Keboola Support team can help you adjust this. 
+In environments with many users, component jobs may queue for Storage job availability, affecting runtime.
+
+## Schedule and Automate
+Within the Flow Builder, click on **Set Schedule** and select when you want the flow to run.
+You can select predefined intervals or set your own. Another option is to use triggers to initiate the run. 
+
+**Scheduling:** Commonly, flows are set to run at specific times. To avoid busy periods in a shared environment, consider scheduling slightly off-peak for smoother execution.
+
+**Triggers:** Set flows to automatically start when certain Storage tables are updated (ideal for managing dependencies across projects). Your projects will stay synchronized and run efficiently.
+
+{: .image-popup}
+![Set Schedule](/tutorial/automate/automate13.png)
+
+## Set Up Notifications
+Once your pipeline or workflow is complete, you may not need to manage it actively every day. Stay on top of your flow's performance by setting up notifications for errors or long run times. From the drop-down 
+list of the project users in the **Notifications** tab, you can select (a) project user(s) who will receive the notification, or you can enter another email address. However, consider using a group email to keep 
+the whole team informed and responsive to any issues.
+
+You can set up an email notification to the following situations: 
+
+- The flow finishes successfully.
+- The flow finishes with warnings; the **Continue on Failure** flag determines whether a task should keep running even if there’s an error. This option is off by default. It is particularly helpful when dealing with APIs that are inconsistent or prone to day-to-day errors. 
```

Comment:
to Continue on Failure bych asi dal někam zvlášť mimo ty notifikace, tady je to takový utopený.
+ by vlastně asi stálo za to explicitně napsat, jak to funguje - tj. fáze je success když jsou všechy joby v ní success a když failne, tak další fáze nepokračuje